src/hb-face.cc

```diff
@@ -79,14 +79,14 @@ hb_face_count (hb_blob_t *blob)
   if (unlikely (!blob))
     return 0;
 
-  /* TODO We shouldn't be sanitizing blob.  Port to run sanitizer and return if not sane. */
-  /* Make API signature const after. */
```

Comment:
Do we still want to make the blob const?

Reply:
Guess not. Looks like the only places we use const in the API for our objects is the `get_user_data` APIs.

Reply:
Actually I'm not sure. Even `hb_blob_get_length` should be const then. Let's not touch.

Reply:
New code still calls functions that don't take const blob. Let's leave it as is.